overlay_gl/overlay.c

```diff
@@ -273,6 +282,17 @@ static void drawOverlay(Context *ctx, unsigned int width, unsigned int height) {
 			return;
 		}
 		ods("Socket connected");
+
+		struct OverlayMsg om;
+		om.omh.uiMagic = OVERLAY_MAGIC_NUMBER;
+		om.omh.uiType = OVERLAY_MSGTYPE_PID;
+		om.omh.iLength = sizeof(struct OverlayMsgPid);
+		om.omp.pid = getpid();
+
+		if (!sendMessage(ctx, &om))
```

Comment:
This is not for every frame.  It's only on socket initialization.

This is from the OS X overlay.  OS X supports the interactive overlay (Mumble client inside the game), and to implement that, it needs the target PID.